### PR TITLE
Hapi v17 compatibility

### PR DESCRIPTION
**Pull request by jamesdixon**

### CODE REVIEW

**Comment by cilindrox** [comment]
@jamesdixon thanks for this, will take a look into it later

**Comment by jamesdixon** [comment]
@cilindrox np! thank you!

**Comment by jamesdixon** [comment]
bump 👍

**Comment by cilindrox** [comment]
aw crap, sorry about that @jamesdixon 

@kuryaki appreciate you're input here, I haven't gotten around Hapi's v17 migration doc

File: lib/index.js [hunk 1]
```diff
@@ -8,46 +8,43 @@ const Hoek = require('hoek');
...
-};
+            // If this is the first connection to Redis on Server start,
+            // wait until it's connected before continuing.
 
-
-exports.register.attributes = {
-    pkg: require('../package.json'),
-    multiple: true
+            // $lab:coverage:off$
+            if (isFirstConnection) {
```
Comment (cilindrox) [inline lib/index.js]: nitpicking here, but what do you think of using `internals.isFirstConnection` instead of `let isFirstConnection`?

**Comment by jamesdixon** [comment]
@cilindrox no worries at all!

File: lib/index.js (hunk 1 above)
Comment (jamesdixon) [reply lib/index.js]: @cilindrox up to you -- it doesn't bother me personally, but whatever gets it merged 👍

**Comment by cilindrox** [comment]
thanks @jamesdixon and apologies for the delay

**Comment by cilindrox** [comment]
published!

**Comment by jamesdixon** [comment]
thank you!

**Pull request merged**